Makefile

```diff
@@ -592,3 +592,9 @@ check-version-warning:
 .PHONY: parse-examples
 parse-examples:
 	go run -tags fields ./hack parseexamples
+
+.PHONY: checksum
+checksum:
```

Comment:
I suggest you do something slightly different. You create the checksum when you create the gz files and same them to a file (e.g. `dist/argo-darwin-foo.gz.shasum256`. 

I think you might want to use openssl for this rather than shasum.

Reply:
Sure thing

Reply:
Done